src/Microsoft.AspNetCore.Authentication.Facebook/FacebookExtensions.cs

```diff
@@ -16,6 +16,9 @@ public static AuthenticationBuilder AddFacebook(this AuthenticationBuilder build
             => builder.AddFacebook(FacebookDefaults.AuthenticationScheme, configureOptions);
 
         public static AuthenticationBuilder AddFacebook(this AuthenticationBuilder builder, string authenticationScheme, Action<FacebookOptions> configureOptions)
-            => builder.AddOAuth<FacebookOptions, FacebookHandler>(authenticationScheme, configureOptions);
+            => builder.AddFacebook(authenticationScheme, authenticationScheme, configureOptions);
```

Comment:
> As soon as you add multiple schemes you'd want unique display names, no?

In 1.0, `Facebook` is always used as the default display name, even if you use a different authentication scheme (so no, it's not unique). I think it would be preferable to have the same behavior in 2.0.